dist/decanter-base.js

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
We don't plan on using these javascript files. I suppose we should add them to the gitignore file. Please remove them from this PR.

Reply:
Removed and added dist/*.js to gitignore